common/playbar.js

```diff
@@ -100,33 +120,7 @@ export function createPlaybar () {
 
     }
 
-    playbarhtml.find("#playbar_tmin").on('input', function() {
-      const tmin = playbarhtml.find("#playbar_tmin");
-      window.animationEngine.activeWindow.backward = Math.abs(Number(tmin.val()));
-      updateClip();
-      window.animationEngine.updateTimeForAll();
-    });
-
-    playbarhtml.find("#playbar_tmax").on('input', function() {
-      const tmax = playbarhtml.find("#playbar_tmax");
-      window.animationEngine.activeWindow.forward = Math.abs(Number(tmax.val()));
-      updateClip();
-      window.animationEngine.updateTimeForAll();
-    });
-
-    playbarhtml.find("#elevation_max").on('input', function() {
-      const elevationMax = playbarhtml.find("#elevation_max");
-      window.elevationWindow[1] = Math.abs(Number(elevationMax.val()));
-    });
-
-    playbarhtml.find("#elevation_min").on('input', function() {
-      const elevationMin = playbarhtml.find("#elevation_min");
-      window.elevationWindow[0] = Math.abs(Number(elevationMin.val()));
-    });
-
-    playbarhtml.find("#myRange").on('input', function() {
-      updateSlider();
-    });
+    playbarhtml.find("#myRange").on('input', updateSlider);
```

Comment:
Was this code not needed / causing bugs? What was the motivation for pulling it out?

Reply:
Unneeded. For elevation_min and _max, the things they are setting, window.elevationWindow[0] and [1], are not referenced anywhere. The (new) ways those effects happen are elsewhere.

Those 4 handlers, for elevation_min and _max, playbar_tmin and _tmax, were rewritten, to not use JQuery. And they were changed to allow positive and negative values, and to go up to the other value (min can go up to current max, and vice versa). But they are still present.

Reply:
Ahh gotcha that makes sense!